site/deno.jsonc

```diff
@@ -0,0 +1,39 @@
+{
+  "lock": false,
+  "nodeModulesDir": "auto",
+  "fmt": {
+    "proseWrap": "preserve"
+  },
+  "tasks": {
+    "setup": "deno task docs:index-versions && deno task docs:genapi",
+    "docs:genapi": "cd api && deno task genapi",
```

Comment:
We should seize the opportunity and merge the two projects into one, but this can happen in a dedicated PQ because it's technically completely unrelated